test123.js

```diff
@@ -1 +1,3 @@
 console.log('Hello World');
+
+console.log('my unique string for realz');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=3387)
```suggestion
console.log("my unique string for realz");
```